docs/source/architecture/servicefees.rst

```diff
@@ -0,0 +1,79 @@
+
+Governance & pricing paper
+---------------------------------------
+
+In order to successfully execute the ``PolicyManager.sol`` contract, the global fee range must be adhered to by the Ursula (and Alice). Failing this, the contract will throw up an error and it will not be possible to commence a commercial engagement or pay/receive fees. Attempts to circumvent NuCypher’s smart contracts are likely to be futile (except perhaps with limited, existing, trust-heavy relationships), given the requirement of coordinated modification and redeployment by network users and a critical mass of other stakers.
+
+Given its high enforceability, the presence of an inflexible fee range dictating the bounds of every transaction is arguably the most critical component of the NuCypher protocol’s economic design and parametrization, particularly over the long-term and with respect to the sustainability of the network. From a governance perspective, it is also amongst the most malleable. If a quorum of stakers wish to set prices outside the range, then they have the right to lobby and propose a widening of the global fee range, its removal altogether, or some other design modification (e.g. narrowing the range). They may do so via the NuCypher DAO – the owner of all NuCypher smart contracts, and controlled by stakers in proportion to their stake size. See <LINK> for guidance on the NuCypher DAO and official NuCypher governance processes.
+
+The **Pricing Protocol & Economics paper** serves as a key resource and reference for community debate, proposals for modification and DAO-driven upgrades and redeployments in the future. The paper discusses the merits and risks of quasi-universal pricing and the enforcement of an upper and lower bound on all offered price points. It includes a price point analysis from a demand-side, service-side and theoretical standpoint to produce the constraints in absolute terms (i.e. in GWEI) that the network will launch with.
+
+
+Setting a discretionary fee rate
+--------------------------------
+
+Stakers should use the ``setMinFeeRate`` function to specify the minimum fee rate that their Ursula (worker machine) will accept.
+
+Note that Alices seeking to instantiate a new sharing policy are able to first discover all current minimum fee rates available to them, by retrieving the list of active stakers’ addresses, then querying the public variable ``PolicyManager.nodes(staker_address).minRewardRate`` with each ``staker_address``.
```

Comment:
```suggestion
Note that Alices seeking to instantiate new sharing policies are able to first discover all current minimum fee rates available to them, by retrieving the list of active stakers’ addresses, then querying the public variable ``PolicyManager.nodes(staker_address).minRewardRate`` with each ``staker_address``.
```

Reply:
Ah, so `PolicyManager.nodes(staker_address).minRewardRate` would return values from stakers who've (naughtily) specified rates outside the range?

Reply:
To set `PolicyManager.nodes(staker_address).minRewardRate`  value - staker must choose within global range, but if global range changes - then `minRewardRate` could be outdated, also before staker even set this value - it'll be zero